modules/core/src/stat.cpp

```diff
@@ -2314,7 +2314,7 @@ void cv::minMaxIdx(InputArray _src, double* minVal,
                     depth == CV_8U ? (ippiMinMaxIndxFuncC1)ippiMinMaxIndx_8u_C1R :
                     depth == CV_8S ? (ippiMinMaxIndxFuncC1)ippiMinMaxIndx_8s_C1R :
                     depth == CV_16U ? (ippiMinMaxIndxFuncC1)ippiMinMaxIndx_16u_C1R :
-                    depth == CV_32F ? (ippiMinMaxIndxFuncC1)ippiMinMaxIndx_32f_C1R : 0;
+                    /*depth == CV_32F ? (ippiMinMaxIndxFuncC1)ippiMinMaxIndx_32f_C1R :*/ 0;
```

Comment:
@vpisarev Is it 32-bit build issue? or 64-bit also?
This patch fixes core test for Win32 and AMD A10-6800K CPU. But this test didn't fail on Win32 build and Intel CPU.